src/_includes/components/usa-step-indicator.njk

```diff
@@ -1,6 +1,5 @@
 <div
   class="usa-step-indicator usa-step-indicator--counters-sm"
-  aria-label="progress"
```

Comment:
Lighthouse complained about this, and I think it was probably right to. This label makes it seem like it's a `progress` element, which this isn't.

Reply:
Non-blocking comment

Oooo interesting, it only flags this on mobile for me.

Noting the error here: 
>[aria-*] attributes do not match their roles

We should flag this for our friends @ USWDS, they're docs use almost this exact code as an example

Reply:
I spent some more time thinking and reading about this and ended up going the other way with it and leaning into the `aria-role="progressbar"` with its other required attributes (e3cc22e). And good call re: the USWDS team. As it is, I don't think the example is correct ARIA usage.